Signal/src/ViewControllers/AppSettings/PrivacySettingsTableViewController.m

```diff
@@ -37,19 +53,20 @@ - (void)updateTableContents
 
     __weak PrivacySettingsTableViewController *weakSelf = self;
 
-    [contents
-        addSection:[OWSTableSection
-                       sectionWithTitle:nil
-                                  items:@[
-                                      [OWSTableItem disclosureItemWithText:
-                                                        NSLocalizedString(@"SETTINGS_BLOCK_LIST_TITLE",
-                                                            @"Label for the block list section of the settings view")
-                                                               actionBlock:^{
-                                                                   [weakSelf showBlocklist];
-                                                               }],
-                                  ]]];
+    OWSTableSection *blocklistSection = [OWSTableSection new];
```

Comment:
Unrelated settings in the privacy view were bunched up into unrelated, confusing clusters.  I've used section headers to break things up a bit.